src/QsCompiler/Tests.Compiler/TestCases/QirTests/TestTargeting.ll

```diff
@@ -3,13 +3,13 @@ entry:
   %q = call %Qubit* @__quantum__rt__qubit_allocate()
   %0 = call %Result* @__quantum__qis__mz(%Qubit* %q)
   %1 = call %Tuple* @__quantum__rt__tuple_create(i64 mul nuw (i64 ptrtoint (i1** getelementptr (i1*, i1** null, i32 1) to i64), i64 2))
-  %2 = bitcast %Tuple* %1 to { %Callable*, %Qubit* }*
-  %3 = getelementptr inbounds { %Callable*, %Qubit* }, { %Callable*, %Qubit* }* %2, i32 0, i32 0
-  %4 = getelementptr inbounds { %Callable*, %Qubit* }, { %Callable*, %Qubit* }* %2, i32 0, i32 1
+  %2 = bitcast %Tuple* %1 to { %Callable*, %Tuple* }*
+  %3 = getelementptr inbounds { %Callable*, %Tuple* }, { %Callable*, %Tuple* }* %2, i32 0, i32 0
+  %4 = getelementptr inbounds { %Callable*, %Tuple* }, { %Callable*, %Tuple* }* %2, i32 0, i32 1
   %5 = call %Callable* @__quantum__rt__callable_create([4 x void (%Tuple*, %Tuple*, %Tuple*)*]* @Microsoft__Quantum__Testing__QIR_____GUID___NoArgs__FunctionTable, [2 x void (%Tuple*, i32)*]* null, %Tuple* null)
   store %Callable* %5, %Callable** %3, align 8
-  store %Qubit* %q, %Qubit** %4, align 8
-  call void @Microsoft__Quantum__ClassicalControl_____GUID___ApplyIfOne__body(%Result* %0, { %Callable*, %Qubit* }* %2)
+  store %Tuple* null, %Tuple** %4, align 8
+  call void @Microsoft__Quantum__ClassicalControl_____GUID___ApplyIfOne__body(%Result* %0, { %Callable*, %Tuple* }* %2)
```

Comment:
Why did this change?

Reply:
The behavior for lifted callables changed to take only a minimal parameter set based on the variables from the calling context that are actually used in the lifted body, whereas before it would just take all known variables as parameters. This change in parameter behavior is reflected here by removing the expected qubit parameter to the lifted callable and replacing it with an empty tuple, as the qubit was not used in the lifted body.